packages/core/src/notification/_notification.scss

```diff
@@ -0,0 +1,5 @@
+@use '~@mezzanine-ui/system/severity';
+
+$prefix: mzn-notif;
+$root-prefix: #{$prefix}-root;
+$severities: severity.$severities-for-feedback;
```

Comment:
renamed after rebase `main` branch
`$severities-for-feedback` => `$severities-wth-info`